src/server/app.py

```diff
@@ -18,8 +19,15 @@
 
 # testing insert
 
-# collection.insert_one({"_id": 0, "user-name":"omar", "user_items": [
+# collection.insert_one({"_id": 0, "user_name":"omar", "user_items": [
 #             {"name":"milk", "category": "dairy",
 #             "purchase_date": "10/6/2020", "expiration_date":"23/6/2020"}
 #       ]
-# })
\ No newline at end of file
+# })
+
+test = collection.find_one({"_id":1})
+@app.route("/")
+def my_index():
+    return flask.render_template("index.html", token=flask.json.loads(flask.json.dumps(test)))
```

Comment:
Just `token=json.dumps(test)` should be fine, I think the flask one is giving you the escaped chars.